src/providers/device/device.ts

```diff
@@ -3,6 +3,8 @@ import { Injectable } from '@angular/core';
 import { Device } from '@ionic-native/device';
 import { AppConfigProvider } from '../../providers/app-config/app-config';
 
+declare let cordova: any;
```

Comment:
What's the reasoning behind `declare let` here?

Reply:
cordova isn't defined, so this is needed to stop TypeScript erroring.